robotbase/src/main/java/frc/robot/controls/DriverControls.java

```diff
@@ -37,7 +37,7 @@ public void mapControls() {
     m_controller
       .a()
       .whileTrue(
-        new DriveToPose(() ->
+        new DriveToPose(Pose ->
```

Comment:
Replace this with an empty pose in the future. I believe there is like a Pose2d.zero or some constant like that.